config/locales/en/claims/schools/users.yml

```diff
@@ -4,3 +4,39 @@ en:
       users:
         index:
           heading: Users
+          add_user: Add user
+          no_users: There are no users for %{school_name}.
+          attributes:
+            users:
+              name:  Name
+              email: Email
```

Comment:
Instead of adding these attributes localisation keys in. Please use `User.human_attribute_name(attribute_name)`. e.g. `User.human_attribute_name(:first_name)` or `User.human_attribute_name(:email)`.

This is create consistency in how we present the attributes names to users.